Build.ps1

```diff
@@ -22,8 +22,8 @@ Process {
     
         Exec { & dotnet clean -c Release }
         Exec { & dotnet build -c Release }
-        Exec { & dotnet test  -c Release --no-build -r "$outputDir" --no-restore -l "trx" -l "console;verbosity=detailed" }
-        Exec { & dotnet pack  -c Release --no-build -o "$outputDir" }
```

Comment:
Required because the Roslyn analyzer is rebuilt and inserted in the main NuGet package during packing